c2corg_ui/templates/article/helpers/view.html

```diff
@@ -31,10 +31,48 @@
       <p><span class="value-title" translate>article_type</span>: <span class="value" x-translate>${article['article_type']}</span></p>
     % endif
 
+    % if article.get('author') and article['article_type'] != 'collab' and article['author']['user_id'] != article['associations']['users'][0]['document_id']:
```

Comment:
Instead of ``article['article_type'] != 'collab'``, why not testing ``article['article_type'] == 'personal'``? It's more straightforward, isn't it?

I don't understand the full condition anyway: if the article is personal, we should show the author's name. Why do you test if ``article['author']['user_id'] != article['associations']['users'][0]['document_id']``?